includes/class-sensei-assets.php

```diff
@@ -310,7 +310,7 @@ private function get_icon_href( string $name ) : string {
 	public function get_icon( string $name, string $class_names = '' ) : string {
 		$href = $this->get_icon_href( $name );
 
-		return '<svg class="' . esc_attr( $class_names ) . '"><use xlink:href="' . esc_url( $href ) . '"></use></svg>';
+		return '<svg class="' . esc_attr( $class_names ) . '"><use href="' . esc_url( $href ) . '"></use></svg>';
```

Comment:
I can left `xlink:href` here too, if we want. Technically, if `href` isn't supported, `xlink:href` can be used as fallback. BUT from what I saw all the browsers released in the last 3+ years support `href` already, so that's why I removed it.